tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-order-mapper-test.php

```
@@ -1055,6 +1055,27 @@ public function test_exception_thrown_when_line_items_empty() {
+					'amount_shipping' => 0,
+					'amount_tax'      => 0,
+					'amount_discount' => 500,
+				],
+			]
+		);
+
+		$this->expectException( Exception::class );
+		$this->expectExceptionMessage( 'discounts are not supported' );
+
+		$this->mapper->create_order_from_checkout_session( $session );
+	}
```

PHPMD (2.15.0)

[error] The method test_exception_thrown_for_discounted_session is not named in camelCase. (undefined)

(CamelCaseMethodName)